src/components/authentication/Login.tsx

```diff
@@ -5,20 +5,31 @@ import Input from "../UI/Input";
 import { MdLockOutline } from "react-icons/md";
 import { BsGoogle } from "react-icons/bs";
 import Card from "../UI/Card";
-import { useNavigate } from "react-router-dom";
+import { Link } from "react-router-dom";
+import axios from "axios";
 
+const url = "/users/login";
 const Login: FC = () => {
-  const [email, setEmail] = useState<string>("");
+  const [username, setUsername] = useState<string>("");
   const [password, setPassword] = useState<string>("");
-  let navigate = useNavigate();
 
   const handleInputPassword = (e: ChangeEvent<HTMLInputElement>) => {
     setPassword(e.target.value);
   };
 
-  const handleSubmitForm = (e: SyntheticEvent) => {
+  const handleSubmitForm = async (e: SyntheticEvent) => {
     e.preventDefault();
-    setEmail("");
+    try {
+      const response = await axios.post(url, { username, password });
+      console.log(response.data);
+    } catch (error) {
+      if (error instanceof Error) {
+        console.log(error.message);
+      } else {
+        console.log("Unexpected error", error);
+      }
+    }
```

Comment:
you can create custom hook to handle the request and error message but this looks fine.

also, cleanup `console.log` and you can use `console.error` for error message in console

Reply:
yes i will do that